app/src/auth/components/AuthLayout/index.jsx

```diff
@@ -11,6 +12,36 @@ type Props = {
     children: React.Node,
 }
 
+const Panel = styled.div`
```

Comment:
This is why text fields for all `auth/` are unstyled. This is their default styling right here.